src/Widgets/Editor.vala

```diff
@@ -43,7 +43,8 @@ public class ENotes.Editor : Gtk.Box {
             code_buffer.text = value.data;
 
             edited = false;
-            ENotes.Headerbar.get_instance ().set_title (value.name);
+            string notebook_title = NotebookTable.get_instance().load_notebook_data(value.notebook_id).name;
+            ENotes.Headerbar.get_instance ().set_title (value.name, notebook_title);
```

Comment:
Can you move these lines out of ENotes.Editor and put them into https://github.com/Philip-Scott/Notes-up/blob/master/src/Widgets/ViewEditStack.vala#L73?

Yes, it was there originally, but I've learned a lot since i first coded this :) Thanks